src/test/kotlin/io/github/mojira/arisa/modules/MultiplePlatformsModuleTest.kt

```diff
@@ -238,7 +238,7 @@ class MultiplePlatformsModuleTest : StringSpec({
         val issue = mockIssue(

```

Comment:
New dungeons config values need to be added to the test module as well.

Reply:
I was just doing that xD